GramAddict/core/log.py

```diff
@@ -59,10 +59,12 @@ def configure_logger():
     global g_log_file_name
     global g_logs_dir
     global g_file_handler
+    global g_log_file_updated
 
     g_session_id = uuid4()
     g_logs_dir = "logs"
     g_log_file_name = f"{g_session_id}.log"
+    g_log_file_updated = False
```

Comment:
I thought about adding it here, but instead I used the variable first_run in __init__.py. If we are using this instead, let's make sure we remove first_run.

Reply:
Yeah, I didnt wanted to concentrated what is related to logging in this file. I intend to create a class to avoid using these globals.